mcodingbot/bot.py

```diff
@@ -9,3 +16,20 @@ def __init__(self) -> None:
 
         self.plugins.load("mcodingbot.plugins.info")
         self.plugins.load("mcodingbot.plugins.reactions")
+
+        self._session: aiohttp.ClientSession | None = None
```

Comment:
I dont like this in bot. YouTubeAPI should be its own object.

Reply:
If its gonna be left here it shouldn't be protected.

Reply:
Its better to use a session property and set `self._session` on `hikari.StartedEvent` or in the start method because thats already overridden imo

Reply:
1 - Then just throw it in the `start` method.
2 - Thats not possible.
3 - Thats not possible anyway because you need an async context to do that. Everything async always starts after started event.

Do what you want though. Im not spending more time bikeshedding XD